pom.xml

```diff
@@ -50,10 +50,8 @@
 
     <developers>
         <developer>
-            <name>Steve Fontes</name>
-            <url>https://github.com/Steve-Fontes</url>
-            <organization>IBM</organization>
-            <organizationUrl>https://www.ibm.com</organizationUrl>
+            <name>MicroProfile Community</name>
```

Comment:
https://github.com/eclipse/microprofile/blob/master/pom.xml#L51

Reply:
I think you can/should leave developers in and mention them individually. So rather than removing Steve, you should add yourself and others. I think there is some kind of status field where you can mark Steve as "retired".
And then your solution is probably more GDPR friendly :)

Reply:
I don't like listing all contributors on a community project. It is often outdated. The primary point of contact should be forum or issues. 

I have added Steve back and added community developer